README.md

```diff
@@ -33,15 +33,33 @@ This repository contains the Helm chart and configuration templates for creating
 
 ### Ingress Management
 
-The observability-platform-api creates **5 separate ingresses** under a unified domain:
+The observability-platform-api creates **6 separate ingresses** under a unified domain:
 
 ```
-https://observability.<codename>.<base-domain>
-├── /loki/api/v1/push          → Gateway Ingress → alloy-gateway (port 3100)
-├── /v1/traces                 → Gateway OTLP    → alloy-gateway (port 4318) 
-├── /loki/api/v1/query*        → Loki Ingress    → loki-gateway (port 80)
-├── /prometheus/api/v1/*       → Mimir Ingress   → mimir-gateway (port 80)
-└── /tempo/api/*               → Tempo Ingress   → tempo-gateway (port 80)
+┌───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+│                              Domain: https://observability.<codename>.<base-domain>                                           │
+├─────────────┬──────────────────────────────────────────────────────────────┬──────────────────────────────────┬───────────────┤
+│ Protocol    │ Path                                                         │ Backend                          │ Type          │
+├─────────────┼──────────────────────────────────────────────────────────────┼──────────────────────────────────┼───────────────┤
+│ HTTP (OTLP) │ /v1/traces                                                   │ observability-gateway-alloy:4318 │ Write         │
```

Comment:
Do we not support grpc here as well?

Reply:
Currently we have [traces ingestion via http](https://github.com/giantswarm/shared-configs/blob/e1bbb58755f00f7cfe86601ce3e3b15a0ca2f59c/default/apps/observability-platform-api/configmap-values.yaml.template#L243), if we want to add support for traces ingestion via gRPC we need to:

- add a new grpc otlp received in alloy gateway with port 4317
- add a the otlp `/opentelemetry.proto.collector.trace.v1.TraceService.Export` path with `observability-gateway-alloy:4317` as backend
---
Sorry my comment did not went through (new github UI glitch)

Reply:
We agreed to no do that now.